test/common/network/utility_test.cc

```diff
@@ -198,6 +198,21 @@ TEST(NetworkUtility, AnyAddress) {
   }
 }
 
+TEST(NetworkUtility, AddressToProtobufAddress) {
+  {
+    envoy::api::v2::Address proto_address;
+    Address::Ipv4Instance address("127.0.0.1");
+    Utility::addressToProtobufAddress(address, proto_address);
+    EXPECT_EQ(true, proto_address.has_socket_address());
```

Comment:
For completeness, mind checking the value and port here, and adding a test case for pipe?